### DISCUSSION TITLE
Bug: cannot apply color temperature (or color) settings via light group

### DISCUSSION THREAD

**Discussion by cerebrate** [jseidl/magic-areas#374] [General]
(This may be the same as #363, but since #363 is flagged as closed I'm raising it anew.)

Basically, a service call which sets the light color temperature, such as this:

```
service: light.turn_on
data:
  kelvin: 6500
target:
  entity_id: light.magic_areas_light_groups_office_overhead_lights
```

fails when applied to the light group, with the error _Failed to call service light.turn_on. two or more values in the same group of exclusion 'Color descriptors' @ data[<Color descriptors>]. Got None_, but succeeds when applied to the lights individually, collectively, or via a non-magic-areas-defined light group. Obviously something of a problem, because having to define parallel manual light groups rather detracts from the point of having light groups in the first place. 

One curiosity I do note is that when [Adaptive Lighting](https://github.com/basnijholt/adaptive-lighting) is configured to operate through the Magic Areas light groups (including the ones in question) it works correctly to adjust color temperature; however, manual service calls as above through the Developer Tools page in HA or service calls embedded in automations fail.

**Comment by cerebrate** [comment]
This is on 4.0.0, BTW.

**Comment by jseidl** [comment]
Weird that that is closed, maybe @mouth4war closed it. This is confirmed.
